.vscode/tasks.json

```diff
@@ -0,0 +1,42 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "build",
+            "command": "dotnet",
+            "type": "process",
+            "args": [
+                "build",
+                "${workspaceFolder}/Chaincase.SSB/Chaincase.SSB.csproj",
+                "/property:GenerateFullPaths=true",
+                "/consoleloggerparameters:NoSummary"
+            ],
+            "problemMatcher": "$msCompile"
```

Comment:
are these vscode tasks universal enough to compile the js and cs?

Reply:
Nah these are accidental junk when I wanted to edit a file quickly.snd opened the dir in vscode